src/js/timingDiagram.js

```diff
@@ -116,7 +116,7 @@ function parseMetadata(data, dataTable, ancestry) {
                 executionCallsCount++;
                 // DA: added the "or now()" safeguard for anything still in Starting or otherwise doesn't have a workflowEnd value, for FireCloud.
                 var endDate = workflowEnd || new Date(Date.now());
```

Comment:
Maybe move this `var` inside the `else` where it's used, or just inline it since it's now only used once.

Reply:
thanks - this is a good suggestion and it led me to investigate further.  In the end I realized the tweak specific to FireCloud wasn't necessary at all - so I just removed it and reverted to using the exact same code Cromwell does (see https://github.com/broadinstitute/cromwell/blob/develop/engine/src/main/resources/workflowTimings/workflowTimings.html).